docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/wysiwyg/common/cgdp-wysiwyg-table/index.ts

```diff
@@ -7,8 +7,8 @@ let _initialized = false;
  */
 const initialize = (): void => {
 	if (!_initialized) {
-		_initialized = true;
 		USATableSortable.createAll();
+		_initialized = true;
```

Comment:
I think this is an artifact of the rebase issue, but this should be reverted back to the original here.